src/ctapipe/instrument/camera/geometry.py

```diff
@@ -672,7 +672,7 @@ def from_table(cls, url_or_table, **kwargs):
 
         version = tab.meta.get("TAB_VER")
         if version not in cls.SUPPORTED_TAB_VERSIONS:
-            raise IOError(f"Unsupported camera geometry table version: {version}")
+            raise OSError(f"Unsupported camera geometry table version: {version}")
```

Comment:
I think this is probably not the correct exception type here (and same in readout.py).  _IOError_ was also probably also misused.

Reply:
True, good catch.

Reply:
We throw IOError everywhere where we cannot load an input file due to an unsupported version.

We could instead throw:

```
class UnsupportedFileVersion(IOError):
    pass
```

?

I'd do that in another PR though

Reply:
Checking the documentation, `IOError` is indeed a deprecated alias of `OSError` and one should use `OSError` directly, even:

```
>>> IOError("foo")
OSError('foo')
```